book/routing.rst

```diff
@@ -1550,6 +1550,18 @@ a template helper function:
             Read this blog post.
         </a>
 
+.. tip::
+
+    If you are generating the route inside a script tag, you might need to properly escape it for Javascript
```

Comment:
We should end the sentence with a colon. And can you please also wrap this paragraph after the first word that crosses the 72nd character?